examples/raspi/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "raspi-oled"
+version = "0.1.0"
+authors = ["chux0519 <chuxdesign@hotmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+embedded-graphics = "0.6.0"
```

```suggestion
embedded-graphics = "0.7.1"
```

although this breaks some stuff in the example code.